estate/models/estate_property.py

```diff
@@ -0,0 +1,52 @@
+from odoo import models, fields
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Table that stores the estate properties"
```

Comment:
```suggestion
    _description = "Estate Property"
```
_description is the human-readable name of the model, not an actual description. It' a misnomer.